30-remote-site-setup.sh

```diff
@@ -140,7 +141,7 @@ echo "condor ALL = ($condor_sudo_users) NOPASSWD: /usr/bin/update-remote-wn-clie
 chmod 644 $CONDOR_SUDO_FILE
 
 grep '^OSG_GRID="/cvmfs/oasis.opensciencegrid.org/osg-software/osg-wn-client' \
-     /var/lib/osg/job-environment*.conf > /dev/null 2>&1 || cvmfs_wn_client=no
+     /var/lib/osg/osg-job-environment*.conf > /dev/null 2>&1 && SKIP_WN_INSTALL=yes || true
```

Comment:
Are you redirecting all output to dev null out of portability concerns for the version of grep installed?

If not, i would use the `-q` and `-s` options instead of the `> /dev/null 2>&1` redirections; eg:
```
 grep -qs '...' && SKIP_WN_INSTALL=yes
```

(`-q` just reports success/failure via the return code, without outputting any lines to stdout, and `-s` does not treat non-existant files as errors -- in this case if your glob pattern doesn't match any files, which presumably is why the `2>&1` is in there.)

But, i realize the way it is predates your changes for this PR, so, the comment is not really directed at you  :)

Also, while the `|| true` (also written `|| :`) is a nice thought to keep in mind, i'll note here that `false && true` (or even `false && false`) does not actually trigger a script to exit under `set -e`. (Unless it's the last line in a function, in which case calling that function will trigger the exit, since the function itself returns failure.)  So, i don't think it's actually necessary here; though it's not wrong either.